examples/wiznet5k_wsgiserver_test.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-FileCopyrightText: 2021 Patrick Van Oosterwijck @ Silicognition LLC
+#
+# SPDX-License-Identifier: MIT
+#
+# This demo was tested with the PoE-FeatherWing, which contains a 24AA02E48
+# chip to provide a globally unique MAC address, but can also work without
+# this chip for testing purposes by using a hard coded MAC.
+#
+# It also contains a `get_static_file` function that demonstrates how to
+# use a generator to serve large static files without using up too much
+# memory.  To avoid having to put extra files in the repo, it just serves
+# `code.py` which isn't very large, but to properly test it, adjust the code
+# to serve an image of several 100 kB to see how it works.
+#
+# There's also an endpoint that demonstrates that `requests` can be used to
+# get data from another socket and serve it.
+#
+
+import board
+import busio
+import digitalio
+import neopixel
+import time
+
+import adafruit_requests as requests
+from adafruit_wiznet5k.adafruit_wiznet5k import WIZNET5K
+import adafruit_wiznet5k.adafruit_wiznet5k_socket as socket
+import adafruit_wiznet5k.adafruit_wiznet5k_wsgiserver as server
+from adafruit_wsgi.wsgi_app import WSGIApp
+
+
+print("Wiznet5k Web Server Test")
+
+
+def get_mac(i2c):
+    "Read MAC from 24AA02E48 chip and return it"
+    mac = bytearray(6)
+    while not i2c.try_lock():
+        pass
+    i2c.writeto(0x50, bytearray((0xFA,)))
+    i2c.readfrom_into(0x50, mac, start=0, end=6)
+    i2c.unlock()
+    return mac
+
+
+def get_static_file(filename):
+    "Static file generator"
+    with open(filename, "rb") as f:
+        bytes = None
+        while bytes is None or len(bytes) == 2048:
+            bytes = f.read(2048)
+            yield bytes
+
+
+# Status LED
+led = neopixel.NeoPixel(board.NEOPIXEL, 1)
+led.brightness = 0.3
+led[0] = (255, 0, 0)
+
```

Comment:
Could you also add the initialization code for the Adafruit FeatherWing Pinout here:
```
# For Adafruit Ethernet FeatherWing
cs = digitalio.DigitalInOut(board.D10)
# For Particle Ethernet FeatherWing
# cs = digitalio.DigitalInOut(board.D5)
spi_bus = busio.SPI(board.SCK, MOSI=board.MOSI, MISO=board.MISO)
eth = WIZNET5K(spi_bus, cs)
```

I tested this locally.

Reply:
Ok, check out the example to see if that works for you?
PoE-FeatherWing and Adafruit Ethernet FeatherWing use the same pinout so it's just called out in the comment.

Reply:
Running on a Feather M4 Express with your updated example throws a `RuntimeError` on the i2c init line:

```
code.py output:
Wiznet5k Web Server Test
Traceback (most recent call last):
  File "code.py", line 67, in <module>
RuntimeError: No pull up found on SDA or SCL; check your wiring

Code done running.```
Hardware: `Adafruit CircuitPython 6.2.0 on 2021-04-05; Adafruit Feather M4 Express with samd51j19`

Reply:
Oh, right.  That makes sense since the FeatherWing provides the pull-ups, not the Feather (wrongly in my opinion).
I fixed it I think for boards that have no I2C pull-ups.  But I don't have any to test so please check it.

Reply:
Tested working on Feather M4+ Ethernet Wing, thanks for making this change.